packages/fxa-payments-server/src/lib/provider.ts

```diff
@@ -0,0 +1,9 @@
+function isStripe(provider: string) {
```

Comment:
(I'm only making this comment because I don't know your level of familiarity with TS.  Ignore if it was a style choice.)  You can do a `export function ...` here and on line 5 and skip line 9.  Or, export the functions and `export default ...` on line 9.

Reply:
thanks - my level of familiarity with TS is what I would definitely consider "entry" so all comments are definitely welcome!